src/index.css

```
@@ -100,6 +100,43 @@ h1 br {
   display: none;
 }
 
+.fallbackWrapper {
+  display: flex;
+  height: 100%;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+}
+
+.spinner-success {
+  color: #28a745;
+}
+
+.spinner-warning {
+  color: #ffc107;
+}
+
+.spinner-danger {
+  color: #dc3545;
+}
+
+@keyframes spinner-border {
+  to { transform: rotate(360deg); }
+}
+
+.spinner-border {
+  display: inline-block;
+  width: 2rem;
+  height: 2rem;
+  vertical-align: text-bottom;
+  border: .25em solid currentColor;
+      border-right-color: currentcolor;
```

Let's fix the indentation here:
```suggestion
  border-right-color: currentcolor;
```